src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0;
```

Comment:
When using BEM, it is recommended not to include external styles such as margins or positions within the BEM block.